packages/nodes-base/nodes/UptimeRobot/UptimeRobot.node.ts

```diff
@@ -36,16 +40,16 @@ export class UptimeRobot implements INodeType {
 		displayName: 'UptimeRobot',
 		name: 'uptimeRobot',
 		icon: 'file:uptimerobot.png',
-		group: ['output'],
+		group: [ 'output' ],
```

Comment:
Codebase convention is no whitespace inside single-line arrays. Same for elsewhere.
```suggestion
		group: ['output'],
```